Modules/Nonunit/Review/include/itkAttributeMorphologyBaseImageFilter.hxx

```diff
@@ -112,18 +112,19 @@ AttributeMorphologyBaseImageFilter<TInputImage, TOutputImage, TAttribute, TFunct
 
   for (RegIt.GoToBegin(); !RegIt.IsAtEnd(); ++RegIt, ++pos)
   {
-    GreyAndPos P;
-    P.Val = RegIt.Get();
-    P.Pos = pos;
-    m_SortPixels[pos] = P;
-    m_Raw[pos] = P.Val;
+    // GreyAndPos P;
+    // P.Val = RegIt.Get();
+    // P.Pos = pos;
```

Comment:
Preferably remove code instead of commenting it out.

Reply:
Thanks - suggested modifications have been made and pushed.